packages/graphql_server/src/updateProject.ts

```diff
@@ -144,6 +144,54 @@ const updateProjectFounders = async (repoName: string, owner: string) => {
   }
 }
 
+const updateProjectForkHistory = async (repoName: string, owner: string) => {
+  if (!(await repoIsAlreadyInDB(repoName, owner))) {
+    return
+  }
+  //@Todo: get real fork history -> waiting for maxis PR
+  const forkHistory = mockForkHistory
+
+  await updateSupabaseProject(repoName, owner, { fork_history: forkHistory })
+}
+
+/**
+ * Updates the github stats of a repo.
+ * Not the star-history though.
+ * @param {string} repoName - The name of the repo.
+ * @param {string} owner - The name of the owner of the repo.
+ */
+const updateProjectGithubStats = async (name: string, owner: string) => {
+  if (!(await repoIsAlreadyInDB(name, owner))) return
+  const githubStats: GitHubInfo | null = await getGithubData(name, owner)
+  if (!githubStats) {
+    console.log('Could not get github stats for ', name, 'owned by', owner)
+    return
+  }
+
+  // *0.9 because the contributor count is not 100% accurate. It is an approximation.
+  // *10 /10 to round to one decimal place
+  //@Todo: refine approximation approach
+  const contributorCount =
+    Math.round(await getContributorCount(owner, name, process.env.GITHUB_API_TOKEN)) * 0.9
+
+  // *10, round, and /10 to round to one decimal place
+  const issuesPerContributor =
```

Comment:
what exactly will *10 do; if it is 4.8 isn't that an accurate number important to investors?

Reply:
Math.round(x *10) / 10 is my way of rounding x to one decimal point. 
for example if x is 4.84 this will return 4.8